MAINTAINERS.md

```diff
@@ -6,58 +6,65 @@
+| Isitha Subasinghe         | [isubasinghe](https://github.com/isubasinghe)           | Reviewer - Workflows                          | [Pipekit](https://pipekit.io/)                  |
+| Jesse Suen                | [jessesuen](https://github.com/jessesuen)               | Lead - CD, Rollouts <br/>Approver - Workflows | [Akuity](https://akuity.io/)                    |
+| Yuan Tang                 | [terrytangyuan](https://github.com/terrytangyuan)       | Lead - Workflows <br/>Reviewer - CD           | [Akuity](https://akuity.io/)                    |
+| William Tam               | [wtam2018](https://github.com/wtam2018)                 | Reviewer - CD                                 | [Red Hat](https://www.github.com/redhat/)       |
+| Julie Vogelman            | [juliev0](https://github.com/juliev0)                   | Approver - Workflows                          | [Intuit](https://www.github.com/intuit/)        |
+| Derek Wang                | [whynowy](https://github.com/whynowy)                   | Lead - Events                                 | [Intuit](https://www.github.com/intuit/)        |
+| Hong Wang                 | [wanghong230](https://github.com/wanghong230)           | Reviewer                                      | [Akuity](https://akuity.io/)                    |
+| Jonathan West             | [jgwest](https://github.com/jgwest)                     | Approver - CD                                 | [Red Hat](https://www.github.com/redhat/)       |
+| May Zhang                 | [mayzhang2000](https://github.com/mayzhang2000)         | Approver - CD                                 | [Intuit](https://www.github.com/intuit/)        |
 
 ## Alumni
 
-| Alumni            | GitHub ID                                   | Project Roles                | Affiliation                                |
-|-------------------|---------------------------------------------|------------------------------|--------------------------------------------|
-| Simon Behar       | [simster7](https://github.com/simster7)     | Approver - Workflows         | [AirBnB](https://www.github.com/airbnb/)   |
-| Xianlu Chen       | [xianlubird](https://github.com/xianlubird) | Reviewer - Workflows         | [Alibaba Cloud](https://github.com/aliyun) |
-| Kareena Hirani    | [khhirani](https://github.com/khhirani)     | Approver - Rollouts          | Independent                                |
-| Mikhail Mazurskiy | [ash2k](https://github.com/ash2k)           | Reviewer - CD                | [GitLab](https://www.github.com/gitlab/)   |
-| Daisuke Taniwaki  | [dtaniwaki](https://github.com/dtaniwaki)   | Approver - Workflows, Events | [Ubie](https://ubie.life/)                 |
+Note: Alumni may also be listed above if they hold approver/reviewer status in some project. Their highest maintainership
+level achieved is recorded below.
```

Comment:
The list of Alumni might get unwieldy if we're going to list all dropped memberships, esp with scoped maintainer and even with the notice that they might be double listed, I think only listing people when they're not considered maintainers at all anymore makes it a bit cleaner,

Reply:
Yeah, I went back and forth on that. The main thing I wanted to avoid was forgetting and then accidentally listing someone at a lower level than they had actually achieved. But that's a small concern.

Reply:
I'm more in favor of having the old roles than not having them. If it's too unwieldy, we can also split out an `EMERITUS.md` file, like `argo-helm` has